src/main/webapp/app/exercises/programming/manage/programming-exercise-detail.component.html

```diff
@@ -76,13 +76,17 @@ <h2><span jhiTranslate="artemisApp.programmingExercise.detail.title">Programming
                         <span jhiTranslate="artemisApp.exercise.teams">Teams</span>
                     </a>
                     <a
-                        *ngIf="programmingExercise.assessmentType === AssessmentType.SEMI_AUTOMATIC || programmingExercise.allowComplaintsForAutomaticAssessments"
+                        *ngIf="
+                            programmingExercise.assessmentType === AssessmentType.SEMI_AUTOMATIC ||
+                            programmingExercise.assessmentType === AssessmentType.MANUAL ||
```

Comment:
Is this even possible? Afaik programming exercises cannot be set to fully manual since there are always automatic test results as a basis.

Reply:
Thanks for your review 😀. As far as I understood from the [Artemis docs assessment page](https://docs.artemis.cit.tum.de/user/assessment/#manual-assessment) manual assessment is available for all exercise types except for the quiz. Please let me know if I misunderstood anything and I will update my PR @Strohgelaender

Reply:
Yes, we call that "manual assessment" in the docs, however, it uses the "SEMI_AUTOMATIC" assessment type internally. This is because the tutors will always start with automatic test results as a basis, true manual assessment (no tests) is not available.

You can verify that by activating manual assessment for a PR and look at the REST response: 
![grafik](https://github.com/ls1intum/Artemis/assets/26540346/606b1d6a-517c-4c59-9dc1-62b10053cead)